build.sbt

```diff
@@ -1,7 +1,8 @@
 val slickVersion = "3.5.1"
 // make sure this is the same as the playWS's dependency
-val pekkoVersion = "1.0.2"
-val playJsonVersion = "3.0.3"
+val pekkoVersion = "1.1.1"
+val playJsonVersion = "3.1.0-M1"
```

Comment:
the latest is 3.0.4

Reply:
@realstraw Do you mind taking another look here?